Issue 8: Facilitate deployment of artifacts to Artifactory via Drone

ascurley opened the pull request:
Rework image to remove the need for an entrypoint script (Drone is likely to override the default entrypoint).

Maven now sources Artifactory credentials directly from the appropriate environment variables.

Update README: add instructions for deploying to Artifactory.

Commit:
Issue 8: Facilitate the deployment of artifacts to Artifactory in Drone pipelines.

Rework image to remove the need for an entrypoint script (Drone is likely to override the default entrypoint).

Maven now sources Artifactory credentials directly from the appropriate environment variables.

Update README: add instructions for deploying to Artifactory.

jaykeshur reviewed (approved):
LG, but can remove "gettext" from the dockerfile too

Commit:
Issue 8: Strip back yum packages added to image.

The following packages are no longer installed:

"curl" (present in base image), "gettext", "unzip" and "wget".
